.ci/azure-pipelines/build-windows.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         displayName: 'Free Up Space'
       - script: |
           mkdir %BUILD_DIR% && cd %BUILD_DIR%
-          cmake $(Build.SourcesDirectory) -G"%GENERATOR%" -DCMAKE_TOOLCHAIN_FILE=%VCPKG_ROOT%\scripts\buildsystems\vcpkg.cmake -DVCPKG_APPLOCAL_DEPS=ON -DPCL_BUILD_WITH_BOOST_DYNAMIC_LINKING_WIN32=ON -DPCL_BUILD_WITH_FLANN_DYNAMIC_LINKING_WIN32=ON -DPCL_BUILD_WITH_QHULL_DYNAMIC_LINKING_WIN32=ON -DBUILD_global_tests=ON -DBUILD_tools=OFF -DBUILD_surface_on_nurbs=ON
+          cmake $(Build.SourcesDirectory) -G"%GENERATOR%" -DCMAKE_BUILD_TYPE="Release" -DCMAKE_TOOLCHAIN_FILE=%VCPKG_ROOT%\scripts\buildsystems\vcpkg.cmake -DVCPKG_APPLOCAL_DEPS=ON -DPCL_BUILD_WITH_BOOST_DYNAMIC_LINKING_WIN32=ON -DPCL_BUILD_WITH_FLANN_DYNAMIC_LINKING_WIN32=ON -DPCL_BUILD_WITH_QHULL_DYNAMIC_LINKING_WIN32=ON -DBUILD_global_tests=ON -DBUILD_tools=OFF -DBUILD_surface_on_nurbs=ON
```

Comment:
Please divide this on multiple lines too (similar to other yaml files, one definition/option per line)

Reply:
It may be that multiline commands are not supported on Windows. Otherwise, I would have written this way when I created the job. But also things may have changed since then. Anyway, please do try to split, this is just a warning.

Reply:
It works now, you just have to use `^` instead of `\` for newlines.